app/src/main/res/layout/fragment_browser.xml

```diff
@@ -20,15 +20,15 @@
         android:layout_height="@dimen/browser_toolbar_height"
         android:visibility="gone" />
 
-    <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+    <mozilla.components.ui.widgets.VerticalSwipeRefreshLayout
```

Comment:
would it be possible to get an overview sometime of this new layout component? Either with an async slack post with a demo or during our weekly show and tell?

Reply:
`VerticalSwipeRefreshLayout` would add better support for gestures like
- scrolling with the finger more horizontally than vertically
- quick scale (zoom with just a finger double tap + swipe)

which I consider to rather be bugs of the platform widget who would show the pull to refresh throbber in these scenarios (when it really shouldn't) (this bugs can be seen in other apps using the platform widget).
The dynamic toolbar would also hook into this functionality and not move for these gestures - better accompanying GeckoView which will also not scroll vertically the webpage.

A bit more details (and before/after videos) can be seen in the 
- ticket - https://github.com/mozilla-mobile/android-components/issues/8185
- PR - https://github.com/mozilla-mobile/android-components/pull/8186

that originally introduced this widget ( 3 1/2 years ago 😲).